src/app/messages/en/messages.json

```diff
@@ -103,7 +103,9 @@
   "i18n:practice-questions:popup:intro:start": "Start now",
   "i18n:practice-questions:popup:navigation:finish": "Finish",
   "i18n:practice-questions:popup:navigation:next": "Next",
+  "i18n:practice-questions:popup:navigation:next:aria-label:after-correct": "This answer is correct. Next button.",
```

Comment:
Is **`after-correct`** in `18n:practice-questions:popup:navigation:next:aria-label:after-correct` necessary? I believe it's about showing some logic like: _this aria-label is added only if the selected answer was correct_. But I'm not sure if logic like that should affect the name.

Reply:
I wanted to make it clear that it is used only after certain event
I've change it to: `i18n:practice-questions:popup:navigation:next:after-submit-correct:aria-label` and `i18n:practice-questions:popup:navigation:show-answer:after-submit-incorrect:aria-label` for Show Answer